Sources/App/Views/PackageController/PackageShow+Types.swift

```diff
@@ -149,22 +152,24 @@ extension PackageShow.Model {
 
     struct PlatformResults: Equatable {
         var ios: BuildResult<PlatformCompatibility>
+        var linux: BuildResult<PlatformCompatibility>
         var macos: BuildResult<PlatformCompatibility>
         var tvos: BuildResult<PlatformCompatibility>
         var watchos: BuildResult<PlatformCompatibility>
-        // TODO: var linux: BuildResult<Platform>
 
         init(iosStatus: BuildStatus,
+             linuxStatus: BuildStatus,
              macosStatus: BuildStatus,
              tvosStatus: BuildStatus,
              watchosStatus: BuildStatus) {
             self.ios = .init(parameter: .ios, status: iosStatus)
+            self.linux = .init(parameter: .linux, status: linuxStatus)
             self.macos = .init(parameter: .macos, status: macosStatus)
             self.tvos = .init(parameter: .tvos, status: tvosStatus)
             self.watchos = .init(parameter: .watchos, status: watchosStatus)
         }
 
-        var cells: [BuildResult<PlatformCompatibility>] { [ios, macos, tvos, watchos] }
+        var cells: [BuildResult<PlatformCompatibility>] { [ios, macos, linux, tvos, watchos] }
```

Comment:
This determines the order of the table columns on the package show page. Ideally those two were more tightly coupled or had the same source but that's not a trivial task due to different aggregation. Probably not worth losing sleep over unless we see a huge influx of platforms.

Reply:
Yea I think this is fine, and I think I agree with the order. The only other option would be to put Linux on the right-hand side as it's quite distinct. I'm happy with it here though.